lib/rubocop/rspec/factory_bot.rb

```diff
@@ -4,7 +4,13 @@ module RuboCop
   module RSpec
     # RuboCop FactoryBot project namespace
     module FactoryBot
-      ATTRIBUTE_DEFINING_METHODS = %i[factory trait transient ignore].freeze
+      ATTRIBUTE_DEFINING_METHODS = %i[
+        factory
+        ignore
+        trait
+        traits_for_enum
+        transient
+      ].freeze
```

Comment:
Adding `:traits_for_enum` caused a `Layout/LineLength` offense, so I turned the horizontal list into a vertical list. I also alphabetized the list to align with what was done with the other constants below.